### ISSUE TITLE
connResetException - even though strong internet connection

### ISSUE THREAD

**Issue by sag333ar** [3speaknetwork/video-encoder#4]
```
5|video  | https://encoder-gateway.infra.3speak.tv
5|video  | jobInfo {"data":{"queueJob":{"reason":"NO_JOBS","job":null}}} { node_id: 'did:key:z6MkuRtVUm7da41KAiKxhEPou4raKkybaD1BJNKgpJoRWnBw' }
5|video  | Error from attempt  Error: aborted
5|video  |     at connResetException (node:internal/errors:692:14)
5|video  |     at TLSSocket.socketCloseListener (node:_http_client:414:19)
5|video  |     at TLSSocket.emit (node:events:539:35)
5|video  |     at node:net:709:12
5|video  |     at TCP.done (node:_tls_wrap:582:7) {
5|video  |   code: 'ECONNRESET'
5|video  | }
5|video  | Error from attempt  Error: aborted
5|video  |     at connResetException (node:internal/errors:692:14)
5|video  |     at TLSSocket.socketCloseListener (node:_http_client:414:19)
5|video  |     at TLSSocket.emit (node:events:539:35)
5|video  |     at node:net:709:12
5|video  |     at TCP.done (node:_tls_wrap:582:7) {
5|video  |   code: 'ECONNRESET'
5|video  | }
5|video  | Error from attempt  Error: Request timed out
5|video  |     at RedirectableRequest.<anonymous> (/Users/sagar/Projects/3speak/video-encoder/node_modules/nodejs-file-downloader/makeRequest.js:31:37)
5|video  |     at RedirectableRequest.emit (node:events:527:28)
5|video  |     at Timeout.<anonymous> (/Users/sagar/Projects/3speak/video-encoder/node_modules/follow-redirects/index.js:164:12)
5|video  |     at listOnTimeout (node:internal/timers:559:17)
5|video  |     at processTimers (node:internal/timers:502:7) {
5|video  |   code: 'ERR_REQUEST_TIMEDOUT'
5|video  | }
5|video  | https://encoder-gateway.infra.3speak.tv
5|video  | jobInfo {"data":{"queueJob":{"reason":"NO_JOBS","job":null}}} { node_id: 'did:key:z6MkuRtVUm7da41KAiKxhEPou4raKkybaD1BJNKgpJoRWnBw' }
5|video  | https://encoder-gateway.infra.3speak.tv
5|video  | jobInfo {"data":{"queueJob":{"reason":"NO_JOBS","job":null}}} { node_id: 'did:key:z6MkuRtVUm7da41KAiKxhEPou4raKkybaD1BJNKgpJoRWnBw' }
5|video  | https://encoder-gateway.infra.3speak.tv
5|video  | jobInfo {"data":{"queueJob":{"reason":"NO_JOBS","job":null}}} { node_id: 'did:key:z6MkuRtVUm7da41KAiKxhEPou4raKkybaD1BJNKgpJoRWnBw' }
5|video  | https://encoder-gateway.infra.3speak.tv
5|video  | jobInfo {"data":{"queueJob":{"reason":"NO_JOBS","job":null}}} { node_id: 'did:key:z6MkuRtVUm7da41KAiKxhEPou4raKkybaD1BJNKgpJoRWnBw' }
5|video  | Error from attempt  Error: aborted
5|video  |     at connResetException (node:internal/errors:692:14)
5|video  |     at TLSSocket.socketCloseListener (node:_http_client:414:19)
5|video  |     at TLSSocket.emit (node:events:539:35)
5|video  |     at node:net:709:12
5|video  |     at TCP.done (node:_tls_wrap:582:7) {
5|video  |   code: 'ECONNRESET'
5|video  | }
5|video  | https://encoder-gateway.infra.3speak.tv
5|video  | jobInfo {"data":{"queueJob":{"reason":"NO_JOBS","job":null}}} { node_id: 'did:key:z6MkuRtVUm7da41KAiKxhEPou4raKkybaD1BJNKgpJoRWnBw' }
5|video  | https://encoder-gateway.infra.3speak.tv
5|video  | jobInfo {"data":{"queueJob":{"reason":"NO_JOBS","job":null}}} { node_id: 'did:key:z6MkuRtVUm7da41KAiKxhEPou4raKkybaD1BJNKgpJoRWnBw' }
5|video  | Error from attempt  Error: aborted
5|video  |     at connResetException (node:internal/errors:692:14)
5|video  |     at TLSSocket.socketCloseListener (node:_http_client:414:19)
5|video  |     at TLSSocket.emit (node:events:539:35)
5|video  |     at node:net:709:12
5|video  |     at TCP.done (node:_tls_wrap:582:7) {
5|video  |   code: 'ECONNRESET'
5|video  | }
5|video  | Error from attempt  Error: aborted
5|video  |     at connResetException (node:internal/errors:692:14)
5|video  |     at TLSSocket.socketCloseListener (node:_http_client:414:19)
5|video  |     at TLSSocket.emit (node:events:539:35)
5|video  |     at node:net:709:12
5|video  |     at TCP.done (node:_tls_wrap:582:7) {
5|video  |   code: 'ECONNRESET'
5|video  | }
5|video  | Final fail Error: aborted
5|video  |     at connResetException (node:internal/errors:692:14)
5|video  |     at TLSSocket.socketCloseListener (node:_http_client:414:19)
5|video  |     at TLSSocket.emit (node:events:539:35)
5|video  |     at node:net:709:12
5|video  |     at TCP.done (node:_tls_wrap:582:7) {
5|video  |   code: 'ECONNRESET'
5|video  | }
5|video  | failing job 845517a9-0e76-4a85-8b3a-44fce980111b
5|video  | Error: aborted
5|video  |     at connResetException (node:internal/errors:692:14)
5|video  |     at TLSSocket.socketCloseListener (node:_http_client:414:19)
5|video  |     at TLSSocket.emit (node:events:539:35)
5|video  |     at node:net:709:12
5|video  |     at TCP.done (node:_tls_wrap:582:7) {
5|video  |   code: 'ECONNRESET'
5|video  | }
```

**Comment by vaultec81** [comment]
Unlike #5 this could be related to the cluster software crashing on the 3speak.tv infrastructure or another networking issue on the 3speak.tv infrastructure side. This is something I've recently noticed unrelated to the video-encoder. I'll be doing more investigation into solving it directly. Regardless, similar strategy as #5 software wise.

**Comment by vaultec81** [comment]
Going to close this in favor of #16 and fixed/mitigated by 31adcc904619c2c2f86fa996f65227ac84f869a0
